extensions/amp-carousel/0.1/amp-carousel.css

```diff
@@ -75,24 +75,28 @@ amp-carousel .amp-carousel-button.amp-disabled {
 
 /* Begin SlideScroll */
 .-amp-slidescroll {
-  overflow: hidden;
-  overflow-x: auto;
-  /*VERTICAL ALIGN - http://zerosixthree.se/vertical-align-anything-with-just-3-lines-of-css/#update*/
-  transform-style: preserve-3d;
-  /*END*/
+}
+
+.-amp-slides-container {
+  height: 100% !important;
+  overflow-x: auto !important;
+  position: relative;
   white-space: nowrap;
+  width: 100% !important;
+  scroll-snap-type: mandatory;
```

Comment:
Yes our compiler supports this out of the box - the vendor prefix is for scrolling not for snap points.